_tags/iteration.md

```diff
@@ -192,6 +192,53 @@ Reverses the order of the loop. Note that this flag's spelling is different from
 6 5 4 3 2 1
 ```
 
+## forloop (object)
+
+Information about a parent [`for` loop](#for).
+
+```json
+{
+  "first": true,
+  "index": 1,
+  "index0": 0,
+  "last": false,
+  "length": 4,
+  "rindex": 3
+}
+```
+
+### Use the `forloop` object
+
+<p class="code-label">Input</p>
+```liquid
+{%- raw -%}
+{% assign smoothie_flavors = "orange, strawberry, banana" | split: ", " %}
+
+{% for flavor in smoothie_flavors -%}
+  {%- if forloop.length > 0 -%}
+    {{ flavor }}{% unless forloop.last %}-{% endunless -%}
+  {%- endif -%}
+{% endfor %}
+{% endraw %}
+```
+
+<p class="code-label">Output</p>
+```text
+orange-strawberry-banana
+```
+
+### forloop (properties)
+
+| Property | Description | Returns |
+| `length` | The total number of iterations in the loop. |  `number` |
```

Comment:
Does there need to be a divider here to separate the headings from the rest of table content? 

`|--- | --- | --- |`